awx/main/tasks/jobs.py

```diff
@@ -146,7 +146,7 @@ def build_execution_environment_params(self, instance, private_data_dir):
         """
         Return params structure to be executed by the container runtime
         """
-        if settings.IS_K8S:
+        if settings.IS_K8S and instance.instance_group.is_container_group:
```

Comment:
:thinking: what about project updates? Their instance group isn't a container group, so it should work by that metric. I would like this `is_container_group` value to be more reliable, or maybe be obvious on `instance` itself.

Anyway, this should work but we need to see it in action with an execution node at some point.